cmd/otelcorecol/go.mod

```diff
@@ -71,10 +70,10 @@ require (
 	github.com/pierrec/lz4/v4 v4.1.22 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
-	github.com/prometheus/client_golang v1.21.1 // indirect
+	github.com/prometheus/client_golang v1.22.0 // indirect
 	github.com/prometheus/client_model v0.6.2 // indirect
-	github.com/prometheus/common v0.62.0 // indirect
-	github.com/prometheus/procfs v0.15.1 // indirect
+	github.com/prometheus/common v0.64.0 // indirect
```

Comment:
I think this leads to problems - this is blocked by https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39729

Reply:
Ah yeah..... interestingly enough i'm not able to repro this failure locally